### PR TITLE
Implemented randomRating() ISSUE : #3

### PR DESCRIPTION
**Pull request by JamesDimi**
Had some problems setting up gulp and running it.( My first ever try to commit on GitHub )
Wrote the function in visual studio code and used .run package to run it and see the results.
The function works as expected, but please try it on the actual server before merging!

### CODE REVIEW

File: server/simulation/random.js [hunk 1]
```diff
@@ -23,7 +23,8 @@ const randomDroneModel = () => {
 };
 
 const randomRating = () => {
-  return 4.8;
+  var rating = (Math.random() * 4) + 1;
```
Comment (TalAter) [inline server/simulation/random.js]: I think `const` would be more appropriate than `var` here.

**Comment by TalAter** [comment]
Looks great!

Just one small change though.

**Comment by TalAter** [comment]
Looks great!
Comments on #3

**Comment by JamesDimi** [comment]
Cool!
Thanks so much for letting me participate and make my first commit!! <3

**Comment by TalAter** [comment]
@JamesDimi Thank you!

Please join our [DAV-Contributors gitter channel](https://gitter.im/DAVFoundation/DAV-Contributors), and hopefully help others looking to get started with open-source and DAV.

**Pull request merged**